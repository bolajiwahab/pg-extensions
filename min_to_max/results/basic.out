CREATE EXTENSION min_to_max;
SELECT min_to_max_config();
 min_to_max_config 
-------------------
 
(1 row)

SELECT pg_sleep(.5);
 pg_sleep 
----------
 
(1 row)

SELECT 1;
 ?column? 
----------
        1
(1 row)

SELECT min_to_max(val) FROM (VALUES(NULL),(NULL)) t(val);
 min_to_max 
------------
 
(1 row)

SELECT min_to_max(val) FROM (VALUES(5),(10),(NULL)) t(val);
 min_to_max 
------------
 5 -> 10
(1 row)

SELECT min_to_max(val) FROM (VALUES(5),(3),(6),(7),(9),(10),(7)) t(val);
 min_to_max 
------------
 3 -> 10
(1 row)

SELECT min_to_max(val) FROM (VALUES('a'),('b'),(' ')) t(val);
 min_to_max 
------------
   -> b
(1 row)

SELECT min_to_max_config('>>');
 min_to_max_config 
-------------------
 
(1 row)

SELECT min_to_max(val) FROM (VALUES(5),(3),(6),(7),(9),(10),(7)) t(val);
 min_to_max 
------------
 3 >> 10
(1 row)

SELECT min_to_max_config();
 min_to_max_config 
-------------------
 
(1 row)

DROP EXTENSION min_to_max;
