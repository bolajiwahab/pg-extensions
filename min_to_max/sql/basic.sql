CREATE EXTENSION min_to_max;
SELECT min_to_max_config();
SELECT pg_sleep(.5);
SELECT 1;
SELECT min_to_max(val) FROM (VALUES(null),(null)) t(val);
SELECT min_to_max(val) FROM (VALUES(5),(3),(6),(7),(9),(10),(7)) t(val);
SELECT min_to_max_config('>>');
SELECT min_to_max(val) FROM (VALUES(5),(3),(6),(7),(9),(10),(7)) t(val);
SELECT min_to_max_config();
DROP EXTENSION min_to_max;
