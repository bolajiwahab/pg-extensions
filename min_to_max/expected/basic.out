CREATE EXTENSION min_to_max;
SELECT min_to_max_config();
 min_to_max_config 
-------------------
 
(1 row)

SELECT pg_sleep(.5);
 pg_sleep 
----------
 
(1 row)

SELECT 1;
 ?column? 
----------
        1
(1 row)

SELECT min_to_max(val) FROM (VALUES(null),(null)) t(val);
 min_to_max 
------------
 
(1 row)

SELECT min_to_max(val) FROM (VALUES(5),(3),(6),(7),(9),(10),(7)) t(val);
 min_to_max 
------------
 3 -> 10
(1 row)

SELECT min_to_max_config('>>');
 min_to_max_config 
-------------------
 
(1 row)

SELECT min_to_max(val) FROM (VALUES(5),(3),(6),(7),(9),(10),(7)) t(val);
 min_to_max 
------------
 3 >> 10
(1 row)

SELECT min_to_max_config();
 min_to_max_config 
-------------------
 
(1 row)

DROP EXTENSION min_to_max;
